doc/ref_impl/cntt-ri2/README.rst

```diff
@@ -0,0 +1,57 @@
+Reference Implementation 2
+==========================
+
+This is the Kubernetes-based Reference Implementation (RI-2)
+
+Release Information
+-------------------
+
+**Bundle: 6.0-alpha**
+
+**Version: 0**
+
+**Release Date: 1st July 2021**
+
+Bundle/Version History
+----------------------
+
+============== ============= ===================
+Bundle.Version Date          Note
+============== ============= ===================
+0.0            15th May 2020 First Initial Draft
+4.0-alpha      25th Sep 2020 Baraque Release
+6.0-alpha      1st July 2021 Kali Release
+============== ============= ===================
+
```

Comment:
do we need to a Lakelse release info

Reply:
yes, that's the problem with keeping these things open for a long time. I'll merge the latest updates. Thanks for the catching this.

Reply:
included in https://github.com/cntt-n/CNTT/pull/2656/commits/e781dee7fd59ba28d36593c292b504b9b2e9d529